src/test/java/com/epam/academy/LanguageServiceTest.java

```diff
@@ -1,68 +1,59 @@
 package com.epam.academy;
 
-import static com.epam.academy.common.Language.HTML;
-import static com.epam.academy.common.Language.JAVA;
-import static com.epam.academy.common.Language.JS;
-import static com.epam.academy.common.Language.KOTLIN;
-import static com.epam.academy.common.Language.PYTHON;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-
-import java.util.Arrays;
-import java.util.Collection;
-
+import com.epam.academy.common.Language;
+import com.epam.academy.common.LanguageService;
+import com.epam.academy.uglyCode.service.LanguageServiceImpl;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 import org.junit.runners.Parameterized.Parameters;
 
-import com.epam.academy.common.Language;
-import com.epam.academy.common.LanguageService;
-import com.epam.academy.uglyCode.service.LanguageServiceImpl;
+import java.util.Arrays;
+import java.util.Collection;
+
+import static com.epam.academy.common.Language.*;
```

Comment:
Wildcard imports are not encouraged. Remember 'YAGNI' principle ? We don't need all those contents in Language class, unless needed

Reply:
Ok got it. I need to recheck my imports every time I ask IntelliJ to auto-import. Sometimes it will change to using wildcard imports.

Reply:
Review the intelliJ setting for imports

Reply:
Got it. Thanks.